docs/examples/use_cases/pytorch/efficientnet/image_classification/training.py

```diff
@@ -211,13 +211,44 @@ def train(
     timeout_handler,
     prof=-1,
     step=0,
+    data_loader_only=False,
 ):
     interrupted = False
 
     end = time.time()
 
     data_iter = enumerate(train_loader)
 
+    if data_loader_only:
+        for i, (input, target) in data_iter:
+            torch.cuda.nvtx.range_push(f"iter {i}")
+
+            bs = input.size(0)
+            lr = lr_scheduler(i)
+            data_time = time.time() - end
+            it_time = time.time() - end
+            log_fn(
+                compute_ips=utils.calc_ips(bs, it_time - data_time),
+                total_ips=utils.calc_ips(bs, it_time),
+                data_time=data_time,
+                compute_time=it_time - data_time,
+                lr=lr,
+                loss=0.0,
+                grad_scale=grad_scale_fn(),
+            )
+
+            torch.cuda.nvtx.range_pop()
+
+            end = time.time()
+            if prof > 0 and (i + 1 >= prof):
+                time.sleep(5)
+                break
+            if ((i + 1) % 20 == 0) and timeout_handler.interrupted:
+                time.sleep(5)
+                interrupted = True
+                break
+        return interrupted
```

Comment:
This is a copy-paste from some other part of the example, right? Can we deduplicate it?